src/olympia/devhub/views.py

```diff
@@ -1267,7 +1267,19 @@ def check_validation_override(request, form, addon, version):
 
 @dev_required
 def version_list(request, addon_id, addon):
-    qs = addon.versions.order_by('-created')
+    unread_count = (
+        (
+            ActivityLog.objects.all()
+            .pending_for_developer(for_version=OuterRef(OuterRef('id')))
+            .filter(versionlog__version=OuterRef('id'))
```

Comment:
Needs a comment explaining that this `filter()` cannot appear before `pending_for_developer()` as that method is doing a subquery, so `OuterRef` would be wrong.